keyboards/skeletonkbd/frost68/info.json

```diff
@@ -0,0 +1,124 @@
+{
+    "manufacturer": "skeletonkbd",
+    "keyboard_name": "Frost68",
+    "maintainer": "skeletonkbd",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+	"matrix_pins": {
+		"cols": [
+			"GP21",
+			"GP22",
+			"GP23",
+			"GP24",
+			"GP25",
+			"GP26",
+			"GP28",
+			"GP3",
+			"GP4",
+			"GP5",
+			"GP6",
+			"GP9",
+			"GP10",
+			"GP11",
+			"GP12",
+			"GP13"
+
+		],
+		"rows": [
+			"GP29",
+			"GP27",
+			"GP20",
+			"GP19",
+			"GP18"
+		]
+	},
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+		"vid": "0x1209",
+		"pid": "0x68A0"
+    },
+    "layouts": {
+        "LAYOUT_alice": {
```

Comment:
Since the PCB only supports a layout, it should be named `LAYOUT` (update the keymaps accordingly)

Reply:
change layout name to 'LAYOUT' 
9e6ecb2c85e65b806950c4e672292631a14883e7

Reply:
There was a fix omission, so I fixed it again.
f2ac0812aa61651fbec42bf2e36429570722d3f2